task/tkn-bundle/0.1/TESTING.md

```diff
@@ -0,0 +1,52 @@
+# Testing tkn-bundle Tekton Task
+
+Make sure you have shellspec installed[1]. Make sure you have access to a
+Kubernetes/OpenShift cluster with Tekton Pipeline running. One such environment
+can be created using `hack/setup-dev-environment.sh`[2] script which brings up a
+kind[3] cluster and installs Tekton Pipeline.
+
+To run the tests run `shellspec` from this directory.
+
+## Example of a testing setup and session
```

Comment:
I'm thinking maybe it's useful to provide another example testing session for the reader who does have their own cluster ready to roll and prefers to use that.

Perhaps it would look like line 35 and onwards, but I'm thinking a typical reader looking at this doc might read "kind cluster" and "hack/scripts from ec-cli", and decide to ignore the rest of the doc.

Reply:
Made the test self-containing